db/models/applications.js

```diff
@@ -77,17 +77,12 @@ class PendingApplication extends Model {
         });
     }
 
-    static findAllByForm(formId, dateRange = {}) {
-        let whereClause = {
-            formId: { [Op.eq]: formId }
-        };
-        if (dateRange.start && dateRange.end) {
-            whereClause = {
-                createdAt: { [Op.between]: [dateRange.start, dateRange.end] }
-            };
-        }
+    static findAllByForm(formId, dateRange) {
```

Comment:
Is there any chance of `req.query` not containing `start` and `end` properties resulting in undefined value for `dateRange` ? (since the default value for dateRange was removed)

Reply:
Default value is always passed in here (only called in one place, and has a default value passed if req.query aren't set), but good shout that maybe the default should be in the method. Will come back to this, but OK to leave it as is for now.